R2Bot/src/Global.cpp

```diff
@@ -3,7 +3,7 @@
 #include <iostream>
 #include <string>
 
-std::string R2Bot::DEVICE_NAME = "";
+std::string R2Bot::DEVICE_NAME = "PI";
```

Comment:
`R2Bot::DEVICE_NAME` is going to be a global variable. We wouldn't want this to be changed here, but instead changed in each `main()` function.